controllers/gitrepository_controller_test.go

```diff
@@ -513,7 +513,7 @@ func TestGitRepositoryReconciler_reconcileSource_authStrategy(t *testing.T) {
 				EventRecorder:               record.NewFakeRecorder(32),
 				Storage:                     testStorage,
 				features:                    features.FeatureGates(),
-				Libgit2TransportInitialized: managed.Enabled,
+				Libgit2TransportInitialized: transport.Enabled,
```

Comment:
Some of the `Libgit2TransportInitialized` logic may no longer be needed now that a) the libgit2 logic is being extracted into an external module b) we only support one state for this.